azplugins/VariantSphereArea.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2018-2019, Michael P. Howard
+// This file is part of the azplugins project, released under the Modified BSD License.
+
+// Maintainer: mphoward
+
+/*!
+ * \file VariantSphereArea.cc
+ * \brief Defines a variant for a shrinking / growing sphere radius.
+ */
+
+#include "VariantSphereArea.h"
+
+namespace azplugins
+{
+/*!
+ * \param R0 Initial radius.
+ * \param alpha Rate of surface reduction (units are area per timestep).
```

Comment:
Should we use "area per timestep" or "area per time" as the unit for alpha? I would have used the latter (if possible), since then alpha does not need to be tuned if the timestep changes.

Reply:
No, this is not possible with the current design of HOOMD. Variants are only aware of the current integer *timestep*, as the (floating-point) size of a timestep is associated to an integrator.